File: HomeWorks/src/Lesson6/Hands/SonyHand.java
package Lesson6.Hands;

public class SonyHand implements IHand {
    private int price;

    public SonyHand() {}

    public SonyHand (int price) {
        this.price = price;
    }

    @Override
    public void upHand() {
        System.out.println("Машет рука Sony");
    }

    public void setPrice(int price) {
        this.price = price;
    }

    @Override
    public int getPrice() {
        return price;
    }
}

File: HomeWorkKotlin/src/Robot/Hands/SonyHand.kt
package Robot.Hands

class SonyHand(private val price_param : Int) : IHand {

    var price = price_param
        set(value) {
            if(value > 0) field = value
        }

    override fun upHand() {
        println("Машет рука Sony")
    }

    override fun getHandPrice(): Int = price
}